activerecord/lib/active_record/migration.rb

```diff
@@ -131,15 +131,15 @@ def initialize(name = nil)
   end
 
   class PendingMigrationError < MigrationError #:nodoc:
+    attr_reader :details
+
     include ActiveSupport::ActionableError
 
     action "Run pending migrations" do
       ActiveRecord::Tasks::DatabaseTasks.migrate
 
       if ActiveRecord::Base.dump_schema_after_migration
-        ActiveRecord::Tasks::DatabaseTasks.dump_schema(
-          ActiveRecord::Base.connection_db_config
-        )
+        ActiveRecord::Tasks::DatabaseTasks.dump_schema(ActiveRecord::Base.connection_db_config)
```

Comment:
I think we should avoid changing this for cosmetic reasons.

Reply:
Whoops, I will fix this -- thanks!